### ISSUE TITLE
Amplify.configure is undefined after upgrading to 2.1

### ISSUE THREAD

**Issue by jfrense** [aws/aws-amplify#228]
I am getting the following error after upgrading aws-amplify to v2.1 - 

`undefined is not an object(evaluating '_awsAmplifyReactNative2.default.configure')
`

It is happening at the following code snippet from my index.js file in my react native project:

```
Amplify.configure({
  Auth: {
      identityPoolId: 'XXX-XXX-XXX-XXX-XXXXX', //REQUIRED - Amazon Cognito Identity Pool ID
      region: 'XXXX', // REQUIRED - Amazon Cognito Region
      userPoolId: 'XXXXX', //OPTIONAL - Amazon Cognito User Pool ID
      userPoolWebClientId: 'XXXXX' //OPTIONAL - Amazon Cognito Web Client ID
  }, Analytics: {
    appId: ' XXXXXX', //OPTIONAL -  Amazon Pinpoint App ID
    region: 'XXXXX', //OPTIONAL -  Amazon service region
}
});
```

**Comment by asharafshahi** [comment]
I just ran into the same thing. I think you need to change your import statement from 'aws-amplify-react-native' to just 'aws-amplify'

**Comment by mlabieniec** [comment]
@jfrense in RN you now use

    import Amplify from 'aws-amplify';

for RN, `aws-amplify-react-natve` is used only for HOCs and components now (same as aws-amplify-react)

**Comment by bijanv** [comment]
@mlabieniec Thanks for this note, can I ask for an update to the docs in that case? I was chasing my tail for a day trying to figure out why this wasn't working when all the examples & docs (see https://docs.aws.amazon.com/aws-mobile/latest/developerguide/react-native-add-storage.html for example or also the AWS Mobile React Native Starter Project - https://github.com/awslabs/aws-mobile-react-native-starter ) are still referencing importing from 'aws-amplify-react-native' . Same with the code example for the amplify mobile starter codebase.

Also would be useful to clarify with examples if / when you would need to utilize aws-amplify-react-native in this case for those not familiar with structure of React's HOCs.

**Comment by mlabieniec** [comment]
@bijanv apologies for the confusion we did update the readme and main guides with this when the release went out. We also updated the rn to console log a deprecated warning when using it. We will go through and make sure all docs are correctly updated it does look like we may have missed this one. Also, for the react native starter there is a pr there in review that will update it to the latest version, for now it is still using the old version. Any updates needed to the docs will get pushed asap, we have a release planned for tomorrow so we will shoot for then.
